viz/com.raytheon.uf.viz.core.maps/localization/bundles/maps/Local Boundaries/Marine_Zones_site.xml

```diff
@@ -34,12 +34,18 @@
                                 lineStyle="SOLID" />
                             <capability xsi:type="colorableCapability"
                                 colorAsString="#9b9b9b" />
+                            <capability xsi:type="labelableCapability"
+                                labelField="id" />
                         </capabilities>
                     </loadProperties>
                     <properties isMapLayer="true" />
                     <resourceData xsi:type="dbMapResourceData">
-                        <table>mapdata.mexico</table>
```

Comment:
Why remove Canada and Mexico? Did we not have the data (ie. map didn't do anything)?  
If we have the data, I think we should leave the options available.  I know a user a little while ago who was looking at NUCAPS data that was mostly up in Canada, and I'm sure the map can help with referencing.

Reply:
The default "State/County" map already includes all other countries. There's nothing added when selecting the Canada or Mexico map

![image](https://github.com/Unidata/awips2-core/assets/65614396/289a0041-2001-4d8e-b0d0-ec455eb01665)

Reply:
That looks like Canada and Mexico are subsets of the State/County Boundaries map.  Something is definitely added when you add Canada or Mexico, as you have them colored green and you can see them in Green on the display.  I agree it might be a bit redundant, but say if you wanted to make an images just like this, that colorizes the United States differently from the rest of North America, you can't do it without the Mexico and Canada maps....
I don't see the advantage of removing some functionality we already have?

Reply:
There is no way to specify just US or North America - all countries/states/counties are displayed within the 

This is what is displayed with just the State/County Boundaries:
![image](https://github.com/Unidata/awips2-core/assets/65614396/03179fc2-2a17-4026-961d-a95eb124c408)

Reply:
Yes I understand that.
But if you were looking at just north america, and wanted to distinguish any of the three countries (Canada, Mexico, or the US), with the Mexico and Canada maps, you can highlight the other country, exactly like you did in the previous image.  Without those maps, you can't do that.

All I'm saying is, they do provide some extra functionality, whether it's really useful or not maybe isn't up to us to decide, and I don't see the advantage of purposely reducing functionality in the software?

Reply:
Yeah, I guess I was trying to make it so 1) the maps menu would all fit on my a laptop screen and 2) it seems odd/random that there are 2 countries that you can load out of the 200? total countries

Reply:
Can they go under the "Other Locations" submenu I thought I saw you added?

Reply:
Not there - that's like Station locations 
![image](https://github.com/Unidata/awips2-core/assets/65614396/f4be5f90-fdc1-4933-b982-163cb536a40f)

Reply:
How about a new submenu "North America"?

Reply:
![image](https://github.com/Unidata/awips2-core/assets/65614396/91b93525-a670-4393-8de7-7ed81ee68865)

Reply:
Okay nice!  Is there an actual US map too? Or is it the State/County Boundaries map?

Reply:
Separate than the State/County, there was a "States" map that is just for the US so I figured I would rename that and move it to the submenu!